### PR TITLE
Release 0.17.0

### PR DESCRIPTION
**Pull request by anton6tak**
# Changes
- #151 added JVM-target support
- #171 append 'toUIColor' usage sample to README
- #157 add info about connect custom buildType to README
- #161 add info about plain file resource access to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -456,6 +474,21 @@ Then just launch task:
...
+After gradle sync we can get file by id `MR.files.test`
+Moko-resources has out of box implementation function for read text files from common code - `readText()`
+
+Usage on Android:
+```
+val text = MR.files.test.getText()
+```
+Usage on Apple:
+```
+val text = FileResource("test", "txt").readText()
```
Comment (Alex009) [inline README.md:488]: why here created resource? here should be `MR.files.test`

File: README.md [hunk 2]
```diff
@@ -456,6 +474,21 @@ Then just launch task:
...
 ```
 
+### Example 10 - plain file resource access
+The first step is a create a resource file `test.txt` for example, in `commonMain/resources/MR/files`
+After gradle sync we can get file by id `MR.files.test`
+Moko-resources has out of box implementation function for read text files from common code - `readText()`
+
+Usage on Android:
+```
+val text = MR.files.test.getText()
```
Comment (Alex009) [inline README.md:484]: where context pass?

**Review by Alex009 — CHANGES_REQUESTED** [review]

**Review by Alex009 — CHANGES_REQUESTED** [review]
readme should be fixed

File: README.md (hunk 2 above)
Comment (anton6tak) [reply README.md]: added

File: README.md (hunk 1 above)
Comment (anton6tak) [reply README.md]: fixed

**Review by Alex009 — APPROVED** [review]

**Pull request merged**